lib/src/main/kotlin/com/swmansion/starknet/data/serializers/JsonRpcErrorPolymorphicSerializer.kt

```diff
@@ -0,0 +1,60 @@
+package com.swmansion.starknet.data.serializers
```

Comment:
I suppose the file name could not include JsonRpc?

Reply:
I think it should, since the naming convention is {class_name}Serializer, and we probably don't wanna rename `JsonRpcError` to just `Error`